core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -316,14 +316,22 @@ sealed abstract class AllowedChartName extends Product with Serializable {
 }
 object AllowedChartName {
   final case object RStudio extends AllowedChartName {
-    def asString: String = "aou-rstudio-chart"
+    def asString: String = "rstudio"
     def trackUsage: Boolean = false
   }
   final case object Sas extends AllowedChartName {
-    def asString: String = "aou-sas-chart"
+    def asString: String = "sas"
     def trackUsage: Boolean = true
   }
-  def stringToObject: Map[String, AllowedChartName] = sealerate.values[AllowedChartName].map(v => v.asString -> v).toMap
+
+  // We used to have different chart names for RStudio and SAS. This is to handle the old names for backwards-compatibility
+  private val deprecatedName: Map[String, AllowedChartName] = Map(
```

Comment:
this serves 2 purposes
1. for backwards-compatibility at contract level. Meaning old value will be accepted
2. for backwards-compatibility at DB level, meaning `getApp` will function properly for old apps that have old chart names